### PR TITLE
update lc docs

### PR DESCRIPTION
**Pull request by marooned20**
Pull Request
============

> Tracked in JIRA by [UP-5123](https://up42.atlassian.net/browse/UP-5123)

### Checklist

- [x] Checked spelling and grammar
- [x] Provided code samples where relevant
- [x] Checked the output of `make html` to ensure new content displays correctly

### CODE REVIEW

**Review by nekhtiari — APPROVED** [review]
Nice!

File: source/up42-blocks/processing/land-cover.rst [hunk 1]
```diff
@@ -19,7 +19,7 @@ Land cover classification or segmentation is the process of assigning each of
 the input imagery pixels a discrete land cover class (e.g. water, forest, urban, desert etc.).
 
 This block is beta version of a land cover classifier trained with Pléiades/SPOT imagery
-in **Sub-Saharan Africa**. The block accepts 4 or 5 as number of classes to be inferred. 
+over **mulitple locations**. The block accepts 4 or 5 as number of classes to be inferred.
```
Comment (perusio) [inline source/up42-blocks/processing/land-cover.rst:22]: typo: multiple

File: source/up42-blocks/processing/land-cover.rst [hunk 2]
```diff
@@ -30,8 +30,8 @@ Currently, classification output consists of following classes:
   5. Urban (including roads and buildings)
 
 Based on an independent test dataset we reached an Accuracy of 0.75 and a Jaccard of 0.22 when trained with 4 classes.
-When trained with 5 classes using the same test dataset, we achieved an Accuracy of 0.64 and a Jaccard of 0.16.
-We recommend that this block is only used within the Sub-Saharan Africa geographic region to achieve optimal results.
+When trained with 5 classes using the same test dataset, we achieved an Accuracy of 0.64 and a Jaccard of 0.51.
```
Comment (perusio) [inline source/up42-blocks/processing/land-cover.rst:33]: Is not present tense better? Communicates action. No?

> When trained with 5 classes using the same test dataset, we achieve an Accuracy of 0.64 and a Jaccard of 0.51.

Can you provide links to give some context to Jaccard? I don't know what it is.

File: source/up42-blocks/processing/land-cover.rst [hunk 3]
```diff
@@ -30,8 +30,8 @@ Currently, classification output consists of following classes:
...
+This block allows now for running inference globally!
```
Comment (perusio) [inline source/up42-blocks/processing/land-cover.rst:34]: No `!` please. Also I can't understand the sentence. Is it supposed to mean that now I can run the inference globally?

**Review by perusio — CHANGES_REQUESTED** [review]
See below please.

File: source/up42-blocks/processing/land-cover.rst (hunk 3 above)
Comment (marooned20) [reply source/up42-blocks/processing/land-cover.rst:34]: Will remove the `!`. 
> Is it supposed to mean that now I can run the inference globally?
precisely!

File: source/up42-blocks/processing/land-cover.rst (hunk 2 above)
Comment (marooned20) [reply source/up42-blocks/processing/land-cover.rst:33]: Fair point. However, based on many research papers, they go for such a style. As this is an experiment and with different datasets and configurations, these numbers might change. It is better to be specific about our experiment and the results we achieved.
As a side remark, I added links for accuracy and Jaccard

**Review by perusio — APPROVED** [review]
LGTM.

**Pull request merged**